src/sentry-global-search/lib/types.ts

```diff
@@ -44,6 +44,13 @@ export type SearchHit = ObjectWithObjectID & {
    */
   _highlightResult?: { [k: string]: { value: string } };
   _snippetResult?: { [k: string]: { value: string } };
+  /**
+   * String representing secondary context information.
```

Comment:
This is an obj now

Reply:
fixed in edcd2d58ebde045da2e32e9d90088b2f9e802432